packages/cli/bin/protosaurus.js

```diff
@@ -52,13 +53,37 @@ const DOCUSAURUS_DIR = process.cwd();
         relativePathToBufGenYaml
       );
 
-      await generator.generate({
-        workDir: pathToBufGenYaml,
-        outPath: `${DOCUSAURUS_DIR}/.protosaurus/generated`
+      // First and foremost, generate cache file.
+      // This is because, `buf ls-files` can identify the new packages/files.
```

Comment:
Can or can't?

Reply:
can, so previously I thought `buf ls-files` only gets updated after it processed the `.proto` files. Turns out it can pick up the new files immediately.

Reply:
But wait! This comment doesn't make sense. I should have deleted it 🤦

Reply:
The context of this comment was, previously I was still on the "2 cache files" thing (1 for buf, 1 for Docusaurus). Now, it would seem that we only require 1 cache file, because `buf ls-files` can immediately pick up the list of files right away before processing them, so yeah.